src/main/resources/META-INF/reaktivity/core.idl

```diff
@@ -114,6 +114,9 @@ scope core
         {
             uint16 length;
             octets[length] payload;
+            uint8 flags = 1;            // 0x01 FIN
+            int64 groupId = 0;
+            int32 claimed = 0;
```

Comment:
May be we don't want to define a default for claimed, but then each component needs to be changed while sending DATA frame

Reply:
Let's put `flags`, `groupId` and `claimed` before `length` so that all fixed length fields come first.

I think we don't want to default `groupId` or `claimed` to avoid mistakes in relaying the correct information from each nukleus implementation.